lib/src/controller.dart

```diff
@@ -371,14 +372,20 @@ class StatefulMapController {
         case GeoJsonFeatureType.polygon:
           final poly = feature.geometry as GeoJsonPolygon;
           for (final geoSerie in poly.geoSeries) {
-            addPolygon(name: geoSerie.name, points: geoSerie.toLatLng());
+            var name = geoSerie.name.isEmpty
+                ? Random().nextInt(10000).toString()
+                : geoSerie.name;
```

Comment:
I think you can declare the `name` variable above to only have one declaration. Moreover instead of generating a fake name using `Random` I'd rather use the [package uuid](https://pub.dev/packages/uuid):
```dart
import 'package:uuid/uuid.dart';

// ...

final name = geoSerie.name.isEmpty
  ? Uuid().v4()
  : geoSerie.name;
```

Reply:
Great ideas. And thinking again about it, I believe the name should be unique. Replacing the name with a UUID would ensure this point.

Reply:
I am quite beginner in Flutter and not 100% sure what you mean by `declaring the variable above to have only one declaration`. So I let you correct my PR, if you want to.

Reply:
Actually I've re-checked the whole code and you can keep the `name` declaration where it is 👍